botuser.php

```diff
@@ -221,8 +222,14 @@
                                 $DB = createDBObject();
                                 checkUserInDB($DB, $event["user"]);
                                 $userinfo = getUserInfo($DB,$event["user"]);
+								$userID = getUserInfo($id);
```

Comment:
What is the variable $id? Where does it come from, and what are you expecting it to contain?

Reply:
`getUserInfo()` is returning an array. You would store like this: `$userInfo = getUserInfo("U3DQ2D81M");` To get the information out of the array, you do something like this `echo($userInfo["id"]);`